src/pages/Selector/container.jsx

```diff
@@ -11,13 +11,19 @@ const SelectorContainer = Wrapped =>
       fetchTemplates().then(templates => this.setState({ templates }));
     }
 
+    handleOnClick = () => {
+      // logic added in next PR
+      console.log('click');
+    };
+
     render() {
       const { employees, templates } = this.state;
 
       return (
         <Wrapped
           employees={transformEmployees(employees)}
           templates={transformTemplates(templates)}
+          handleOnClick={() => this.handleOnClick()}
```

Comment:
So this one here we can chat in person, this would be better if the prop was called onClick and pass the function reference to it? `handleOnClick={() => this.handleOnClick}`